torch/linalg/__init__.py

```diff
@@ -536,9 +535,6 @@
 
 .. note:: For CUDA inputs, this function synchronizes that device with the CPU.
 
-.. note:: This function is not differentiable. If you need differentiability use
-          :func:`torch.linalg.eigh` instead, which also computes the eigenvectors.
-
```

Comment:
Add a `seealso:` section now, given that there is no explicit reference to `eigh`. E.g.
```
.. seealso::
          :func:`torch.linalg.eig` computes the full eigenvalue decomposition. 
```